docs/design/chart_format.md

```diff
@@ -189,25 +176,14 @@ keywords:
 * proxy
 source: https://github.com/foo/bar
 home: http://nginx.com
-depends:
-        charts:
-          - name: memcached
-                location: helm.sh/charts/memcached
-                version: 0.9.3
-        kubernetes:
-                version: “>=1.0.0”
-                extensions:
-                  - extensions/v1beta1
+expander:
+  name: GoTemplates
+schema: Schema.yaml
 ```
 
-#### Dependency Information
-The chart format allows the specification of _runtime dependencies_ (not build-time dependencies). For example, a Wordpress chart might have a runtime dependency on a MySQL database. The database needn’t be present for processing the template file, but must be present in the cluster in order for Wordpress to function.
-
+### Expanders and Templates
+The content of the `templates/` directory and the schema file are defined by the particular expander invoked by name in the Chart.yaml file.  Such expanders consume these files, in the context of properties, to generate content.  If a schema is given, the expander may use it to validate the properties before expansion.  Discussion of the available expanders and how they intepret the content of /templates is outside the scope of this document.
 
-Dependencies hold between charts. That is, one chart may depend upon another chart.
-
-
-The resolution of dependencies is not covered in this proposal, as it is an implementation detail of the DM service.
 ### Releasing a Chart
 A chart is _released_ when the source of the chart is tested, versioned, packaged into a gzipped tar file, and provenanced. Testing and provenancing attach badges to the Chart that attest to its quality and provenance. At that point, that particular release of a chart is considered immutable. No further changes are allowed. To enforce immutability through tamper detection, Charts must be digitally signed.
 
```

Comment:
Let's use `gs://kubernetes-charts` instead of `https://helm.sh/charts` in the example below, since we don't actually support the latter in the code.